src/documentation/components/ThemeSwitcher.vue

```diff
@@ -80,24 +80,68 @@
 
 <script lang="ts">
 import { Themes } from '../models/models';
-import { THEMES } from '../constants/constants';
+import { THEMES, defaultCss, defaultDocsCss } from '../constants/constants';
 import { Component, Vue } from 'vue-property-decorator';
 
 declare const chi: any;
+interface AssetsToReplace {
+  type: 'css' | 'docsCss';
+  id: string;
+}
 
-@Component({})
+@Component({
+  head() {
+    return {
+      link: [
+        {
+          rel: 'stylesheet',
+          id: 'chi-css',
+          type: 'text/css',
+          href: defaultCss
+        },
+        {
+          rel: 'stylesheet',
+          type: 'text/css',
+          id: 'chi-docs-css',
+          href: defaultDocsCss
+        },
+      ],
+    }
+  }
+})
 export default class ThemeSwitcher extends Vue {
-  theme: Themes = this.$store.state.themes.theme;
   themes = THEMES;
 
-  setTheme(theme: Themes) {
+  setTheme(theme: Themes): void {
+    const brandLogo = document.getElementById('header-logo') as any;
+    const assetsToReplace : AssetsToReplace[] = [{type: 'css', id: 'chi-css'}, {type: 'docsCss', id: 'chi-docs-css'}];
+
+    assetsToReplace.forEach((asset: { type: 'css' | 'docsCss', id: string }) => {
```

Comment:
Could this be replaced with `forEach((asset: AssetsToReplace)` ?

Reply:
Yes, good catch